exporter/importables.go

```diff
@@ -1951,4 +2002,75 @@ var resourcesMap map[string]importable = map[string]importable{
 			// {Path: "http_url_spec.authorization", Variable: true},
 		},
 	},
+	"databricks_access_control_rule_set": {
+		AccountLevel: true,
+		Service:      "access",
+		// Name: func(ic *importContext, d *schema.ResourceData) string {
+		// 	return "webhook_" + d.Id()
+		// },
+		List: func(ic *importContext) error {
+			accountId := ic.Client.Config.AccountID
+			// emit default ruleset
+			ic.Emit(&resource{
+				Resource: "databricks_access_control_rule_set",
+				ID:       fmt.Sprintf("accounts/%s/ruleSets/default", accountId),
+			})
+			return nil
+		},
+		Import: func(ic *importContext, r *resource) error {
+			var rule iam.RuleSetResponse
+			s := ic.Resources["databricks_access_control_rule_set"].Schema
+			common.DataToStructPointer(r.Data, s, &rule)
+
+			for _, grant := range rule.GrantRules {
+				for _, principal := range grant.Principals {
+					parts := strings.Split(principal, "/")
+					if len(parts) != 2 {
+						log.Printf("[WARN] Incorrect principal: '%s'", principal)
+						continue
+					}
+					switch parts[0] {
+					case "users":
+						ic.Emit(&resource{
+							Resource:  "databricks_user",
+							Attribute: "user_name",
+							Value:     parts[1],
+						})
+					case "servicePrincipals":
+						ic.Emit(&resource{
+							Resource:  "databricks_service_principal",
+							Attribute: "application_id",
+							Value:     parts[1],
+						})
+					case "groups":
+						ic.Emit(&resource{
+							Resource:  "databricks_group",
+							Attribute: "display_name",
+							Value:     parts[1],
+						})
+					default:
+						log.Printf("[WARN] Unknown principal type: '%s'", parts[0])
+					}
+				}
+			}
+
+			return nil
+		},
+		Depends: []reference{
+			{Path: "grant_rules.principals", Resource: "databricks_user", Match: "acl_principal_id"},
+			{Path: "grant_rules.principals", Resource: "databricks_group", Match: "acl_principal_id"},
+			{Path: "grant_rules.principals", Resource: "databricks_service_principal", Match: "acl_principal_id"},
+			{Path: "name", Resource: "databricks_service_principal", Match: "application_id", MatchType: MatchRegexp,
+				Regexp: regexp.MustCompile("^accounts/[^/]+/servicePrincipals/([^/]+)/ruleSets/default$")},
+			{Path: "name", Resource: "databricks_group", MatchType: MatchRegexp,
+				Regexp: regexp.MustCompile("^accounts/[^/]+/groups/([^/]+)/ruleSets/default$")},
```

Comment:
we also support the account rule set btw - `accounts/{accountId}/ruleSets/default`. Should that be included here?

Reply:
These are replacement rules that convert something like `accounts/111111/servicePrincipals/123456/ruleSets/default` into Terraform code with reference to another resource `accounts/111111/servicePrincipals/${databricks_service_principal.123456.application_id}/ruleSets/default`.

Default ruleset is handled in the actual export logic [here](https://github.com/databricks/terraform-provider-databricks/pull/2699/files#diff-ab7238b14b515ceaca6b7f28691827b1044cb9a521f517753a17aa87b20c795bR2014).